diffkemp/simpll/Utils.cpp

```diff
@@ -686,6 +686,34 @@ std::string valueToString(const Value *Val) {
     return DumpStrm.str();
 }
 
+Constant *getConstantFromString(Constant *initializer, std::string Val) {
```

Comment:
Why does this function use initializer? Just to get the variable type? In such case I think that it would be better to pass the type directly.